server/react-admin/src/AdminApp.js

```diff
@@ -1,114 +1,109 @@
 import React from 'react'
-import { BrowserRouter, Route, Switch } from 'react-router-dom'
+import { styled } from '@mui/material/styles'
+import { Route, Switch, useLocation } from 'react-router-dom'
+import SideNav from './sidemenu/SideNav'
 import AdminHomeRoute from './adminHomeRoute/AdminHomeRoute'
-import CategoryRoute from './categories/CategoryRoute'
-import CategoryForm from './categories/CategoryForm'
 import PublicationRoute from './publications/PublicationRoute'
 import PublicationForm from './publications/PublicationForm'
 import VideoTestimonialRoute from './videoTestimonials/VideoTestimonialRoute'
 import VideoTestimonialForm from './videoTestimonials/VideoTestimonialForm'
 import AdvocacyGroupsRoute from './advocacyGroups/AdvocacyGroupsRoute'
 import AdvocacyGroupsForm from './advocacyGroups/AdvocacyGroupsForm'
```

Comment:
Needs to be updated so it doesn't delete the routes to category, levels, geography.